java/acorn-openai/src/test/java/com/datasqrl/ai/trace/TraceJudgeTest.java

```diff
@@ -0,0 +1,38 @@
+package com.datasqrl.ai.trace;
+
+import com.datasqrl.ai.trace.QualitativeTraceJudge.QualitativeResult;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import lombok.SneakyThrows;
+import org.apache.commons.configuration2.MapConfiguration;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Disabled;
+import org.junit.jupiter.api.Test;
+
+public class TraceJudgeTest {
```

Comment:
@mbroecheler Why is this test in the openai module? Shouldn't it rather be in the trace module? The judge model is quite irrelevant for the test.